build/build.js

```diff
@@ -1,5 +1,6 @@
 'use strict';
 
+const _ = require('lodash');
```

Comment:
let's avoid lodash

Reply:
Done, but why we don't like lodash?

Reply:
mostly because i don't want node code to end up like browser-side code and the tendency to pull in mega-libraries like lodash or jquery